app/lib/features/chat/pages/room_page.dart

```diff
@@ -91,15 +88,55 @@ class _RoomPageConsumerState extends ConsumerState<RoomPage> {
     );
   }
 
+  void onBackgroundTap() {
+    var emojiRowVisible = ref.read(
+      chatInputProvider.select((ci) {
+        return ci.emojiRowVisible;
+      }),
+    );
+    var roomNotifier = ref.read(chatRoomProvider.notifier);
+    var inputNotifier = ref.read(chatInputProvider.notifier);
+    if (emojiRowVisible) {
+      roomNotifier.currentMessageId = null;
+      inputNotifier.emojiRowVisible(false);
+    }
+  }
+
+  Widget avatarBuilder(String userId) {
+    final memberProfile = ref.watch(memberProfileProvider(userId));
```

Comment:
I am skeptical if this would ever work and be reactive, try wrapping this whole method with ```Consumer``` builder.

Reply:
Also this function is using `ref.watch()`.
https://github.com/acterglobal/a3/blob/5c8b75455d40972db2cf23ab2afd8d2c615b8f5f/app/lib/features/chat/pages/room_page.dart#L94-L105
Should it be wrapped by `Consumer()`?

Reply:
Fixed by https://github.com/acterglobal/a3/pull/855/commits/4239d5e28ce9904e7638301fb7f94b861d1481ae